README.md

```diff
@@ -1 +1,178 @@
 # ng-bem-components
+
+An [AngularJS](angularjs.org) `angular-bem` module for using components based on [`i-bem.js`](https://bem.info/libs/bem-core/v2.7.0/desktop/i-bem/) framework.
+See [bem-core](https://github.com/bem/bem-core) and [bem-components](https://github.com/bem/bem-components) for further details.
+
+Module will allow you to render `BEMJSON`, help you to work with BEM mods and event, and provide some integrations of `i-bem` blocks with Angular directives.
+
+## Install
+
+```sh
+bower i ng-bem-components --save
+```
+
+`angular-bem` depends on `i-bem__dom`, `jquery`, `BEMHTML` and `BEMTREE` (optional) from `ymodules` modular system.
+Include `angular-bem.js` in your js bundle or html file in any place where `ymodules` already defined.
+
+## Bootstrap Angular app
+
+You will not able to use `ngApp` directive since dependencies provided asynchronously.
```

Comment:
will not able -> will not be able
dependencies provided -> dependencies are provided